frontend/src/routes/Search.tsx

```diff
@@ -585,7 +599,145 @@ const SearchSeries: React.FC<SearchSeriesProps> = ({ id, title, description, hos
                 />}
             </div>
         </WithIcon>
-    </Item>;
+        <ThumbnailStack {...{ thumbnailInfo, title }} />
+    </Item>
+;
+
+type ThumbnailStackProps = {
+    title: string;
+    thumbnailInfo: readonly {
+        readonly audioOnly: boolean;
+        readonly isLive: boolean;
+        readonly thumbnail: string | null | undefined;
+    }[] | undefined;
+}
+
+const ThumbnailStack: React.FC<ThumbnailStackProps> = ({ thumbnailInfo, title }) => {
+    const key = useId();
+
+    return <div css={{
+        ...thumbnailCss,
+        outline: 0,
+        display: "grid",
+        gridAutoColumns: "1fr",
+        "> div:not(:last-child)": {
+            boxShadow: "3px -2px 8px rgba(0, 0, 0, 20%)",
+        },
+        "> div:nth-child(1)": {
+            zIndex: 3,
+            gridColumn: "1 / span 10",
+            gridRow: "3 / span 10",
+        },
+        "> div:nth-child(2)": {
+            zIndex: 2,
+            gridColumn: "2 / span 10",
+            gridRow: "2 / span 10",
+        },
+        "> div:nth-child(3)": {
+            zIndex: 1,
+            gridColumn: "3 / span 10",
+            gridRow: "1 / span 10",
+        },
+    }}>
+        {
+            thumbnailInfo?.map((info, idx) => <div key={key + idx} css={{
```

Comment:
[`useId` shouldn't be used for this purpose](https://react.dev/reference/react/useId#usage). Using just `idx` as `key` here is fine as the array is only ever 3 items long.